### DISCUSSION TITLE
Screw position for left side thumb cluster is symetric to that of left side

### DISCUSSION THREAD

**Discussion by adegorre** [joshreve/dactyl-keyboard#44] [General]
It took me some time to find out why the thumb cluster screw hole was always somewhere outside the walls of the case, and that did not depend on the thumb cluster shape (all of this time, I was looking at left side only).
I finally understood, that there was no different handling of the two sides: the function `screw_insert_thumb` decides on the shape only reading the variable `thumb_style` in all cases (which only gives the style of the right hand side thumb cluster) and never `other_thumb`.

Am I doing something wrong or is it a bug, as it looks like?

**Comment by joshreve** [comment]
It was totally a bug, but I thought I fixed that.  I may have done it in the dev branch but haven't merged yet.  What branch and commit are you using?  That will help me figure out how to direct you get it fixed.  I do know exactly what you are talking about so I should be able to get it running correctly for you quickly.
